src/utils/history.js

```diff
@@ -0,0 +1,3 @@
+import { createBrowserHistory } from "history";
+
+export default createBrowserHistory();
```

Comment:
what will this be used for?

Reply:
This was part of the react-router tracking. When you incorporate it with GA it will trigger an event every time the url changes in the browser and should log the new page. This is only a partial implementation and the guide is posted in the comments on app.js page.